### DISCUSSION TITLE
Add ability to see all loan return incidents from a loan in a grid

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5567] [Feature Requests]
Requested by Edinburgh
Currently, you can only see returns by going to the item loaned and clicking the Loan return button, but that only displays the return info for that item.  They would like to see the loan return information for all of the items in one view

**Comment by maxpatiiuk** [comment]
Fixed on `testability` by allowing to change the SubView from Form to Grid format (using the Form Preferences) dialog
